lib/match/git_helper.rb

```diff
@@ -1,10 +1,12 @@
 module Match
   class GitHelper
-    def self.clone(git_url)
+    def self.clone(git_url, shallow_clone)
```

Comment:
What's the recommended code style to add parameters to existing methods ?

Maybe ?

``def self.clone(git_url, shallow_clone: true)``